api/k8s/v1beta4/k8s_helpers.go

```diff
@@ -8,9 +8,14 @@ import (
 	"github.com/scaleway/scaleway-sdk-go/scw"
 )
 
+var (
+	RetryInterval = defaultRetryInterval
+)
+
 const (
 	waitForClusterDefaultTimeout = time.Minute * 15
 	waitForPoolDefaultTimeout    = time.Minute * 15
+	defaultRetryInterval         = 5 * time.Second
```

Comment:
```suggestion
	defaultRetryInterval         = time.Second * 5
```

for consistency with line above